elastalert/elastalert.py

```diff
@@ -1347,6 +1349,14 @@ def send_alert(self, matches, rule, alert_time=None, retried=False):
                 kb_link_formatter = self.get_kibana_discover_external_url_formatter(rule)
                 matches[0]['kibana_discover_url'] =  kb_link_formatter.format(kb_link)
 
+        if rule.get('generate_opensearch_discover_url'):
+            kb_link = generate_opensearch_discover_url(rule, matches[0])
+            if kb_link:
+                kb_link_formatter = self.get_opensearch_discover_external_url_formatter(rule)
```

Comment:
Please use a variable name other than kb_link_formatter. Used by kibana discovery

Reply:
done, renamed to opsh_*